cmd/tracee-ebpf/internal/flags/flags-containers.go

```diff
@@ -0,0 +1,81 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/containers/runtime"
+)
+
+func ContainersHelp() string {
+	return `Select which container runtimes to connect to for container events enrichment.
+By default, if no flag is passed, tracee will automatically detect installed runtimes by going through known runtime socket paths.
+If runtimes are specified, only the ones passed through flags will be connected to through the provided socket file path.
+Supported runtimes are:
+
+1. CRI-O (crio)
+2. Containerd (containerd)
+3. Docker (docker)
+
+Example:
+  --containers crio:/var/run/crio/crio.sock
+`
+}
+
+func contains(s []string, val string) bool {
+	for _, str := range s {
+		if str == val {
+			return true
+		}
+	}
+	return false
+}
+
+func PrepareContainers(containerFlags []string) (runtime.Sockets, error) {
+	if len(containerFlags) == 0 {
+		return autodiscoverSockets(), nil
+	}
+
+	supportedRuntimes := []string{"crio", "containerd", "docker"}
+
+	sockets := runtime.Sockets{}
+
+	for _, flag := range containerFlags {
+		parts := strings.Split(flag, ":")
+		if len(parts) != 2 {
+			return sockets, fmt.Errorf("failed to parse container flags (must be of format {runtime}:{socket path})")
+		}
+		containerRuntime := parts[0]
+
+		if !contains(supportedRuntimes, containerRuntime) {
+			return sockets, fmt.Errorf("provided unsupported container runtime (see --containers help for supported runtimes)")
+		}
+
+		socket := parts[1]
+		err := sockets.Register(containerRuntime, socket)
+
+		if err != nil {
+			return sockets, fmt.Errorf("failed to register runtime socket, %s", err.Error())
+		}
+	}
+
+	return sockets, nil
+}
+
+//check default paths for all supported container runtimes and aggregate them
+func autodiscoverSockets() runtime.Sockets {
+	sockets := runtime.Sockets{}
+	const (
+		defaultContainerd = "/var/run/containerd/containerd.sock"
+		defaultDocker     = "/var/run/docker.sock"
+		defaultCrio       = "/var/run/crio/crio.sock"
+		defaultPodman     = "/var/run/podman/podman.sock"
```

Comment:
Not yet but i'm keeping the socket for future use, because we will want to support podman. Maybe we will get it in this PR as well. I can remove it in the end if we won't.

Reply:
ok sounds good. Let's just remember to actually remove it if not used, as you said.